share/python/qcdb/molecule.py

```diff
@@ -303,7 +303,9 @@ def save_string_xyz(self, save_ghosts=True):
             for i in range(self.natom()):
                 if self.Z(i):
                     N += 1
-        text = "%d\n" % (N)
+        text = ''
+        if save_natom:
+            text += "%d\n" % (N)
```

Comment:
So very true. And that was my first attempts to reconcile the psi4.Mol and qcdb.Mol functions. But the fact is that this fn is mostly used to generate a string to plop within `molecule mymol {...}`, and in that context the `N` is trouble. So better to leave psi4 alone and alter qcdb.